src/views/List.vue

```diff
@@ -1,40 +1,54 @@
 <template>
   <div>
+    <br>
     <h1>List</h1>
     <p>{{list}}</p>
     <p>{{list.listItems}}</p>
-    <v-data-table
-      :headers="headers"
-      :items="list.listItems"
-      :items-per-page="5"
-      class="elevation-1"
-    ></v-data-table>
+    <ol style="list-style-type:none;">
+      <li v-for="(item,i) in listItems" :key="`${item.text}${i}`">
```

Comment:
Use a more descriptive variable that `i`, such as `idx` (shorthand for "index") or just `index`